mindmaps-graql/src/main/java/io/mindmaps/graql/internal/parser/MatchQueryPrinter.java

```diff
@@ -19,35 +19,37 @@
 package io.mindmaps.graql.internal.parser;
 
 import io.mindmaps.core.model.Concept;
-import io.mindmaps.graql.MatchQueryDefault;
+import io.mindmaps.graql.MatchQuery;
 
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 import java.util.stream.Stream;
 
 /**
  * A printer for a MatchQuery, that uses a map of Getters to decide what information to print
  */
 public class MatchQueryPrinter {
```

Comment:
It's only used by the parser, to handle some parse-exclusive behaviour (all the stuff that changes the format of the output, like `select $x(has name)` or whatever)